internal/connection_manager.go

```diff
@@ -29,30 +28,33 @@ const (
 	serializationVersionMismatch
 )
 
-type connectionManager struct {
-	client              *HazelcastClient
-	connections         map[string]*Connection
-	lock                sync.RWMutex
-	nextConnID          int64
-	connectionListeners atomic.Value
-	listenerLock        sync.Mutex
-}
+type connectionManager interface {
+	//returns a snapshot of active connections
+	getActiveConnections() map[string]*Connection
```

Comment:
the convention of documentation is `method name ....` so they should be in the form of `getActiveConnections returns ...`

Reply:
ok, I will change this